test/files/run/t1931.check

```diff
@@ -10,8 +10,8 @@ import Predef.{any2stringadd=>_, _}
 
 scala> x + " works"
 <console>:14: error: value + is not a member of Any
-       x + " works"
-         ^
+x + " works"
+  ^
```

Comment:
Noticed that this is a regression. The text is supposed to align with text at prompt. I haven't checked the current head.